internal/cmdliveinit/cmdliveinit_test.go

```diff
@@ -294,7 +294,7 @@ func TestCmd_Run(t *testing.T) {
 			// Otherwise, validate the kptfile values and/or resourcegroup values.
 			var actualInv kptfilev1.Inventory
 			assert.NoError(t, err)
-			kf, err := pkg.ReadKptfile(filesys.FileSystemOrOnDisk{}, w.WorkspaceDirectory)
+			kf, err := pkg.ReadKptfile(types.DiskPath(w.WorkspaceDirectory))
```

Comment:
This is more of a note. Eventually, I would like code that reads `Kptfile` to first initialize the pkg and then read kptfile using pkg API like we are doing in most places.